apps/electron/layers/main/src/handlers/db/ensure-db.ts

```diff
@@ -0,0 +1,89 @@
+import { watch } from 'chokidar';
```

Comment:
This project has not been maintained for a long time. I'm worried about this

https://github.com/paulmillr/chokidar

Why don't we use Node.js API?

Reply:
I tried to use Node.js api first, but it does not work pretty well.

On the README of chokidar, it listed some issues:

```
Why?
Node.js fs.watch:

Doesn't report filenames on MacOS.
Doesn't report events at all when using editors like Sublime on MacOS.
Often reports events twice.
Emits most changes as rename.
Does not provide an easy way to recursively watch file trees.
Does not support recursive watching on Linux.

Node.js fs.watchFile:

Almost as bad at event handling.
Also does not provide any recursive watching.
Results in high CPU utilization.
Chokidar resolves these problems.
```

I do encounter most of the issues when using either `fs.watch` or `fs.watchFile`. For a more reliable implementation, I used the popular chokidar instead.

BTW do you have any other suggestions over chokidar?
I saw in https://github.com/parcel-bundler/parcel/issues/863 it listed some alternatives like: `watchman`, `fb-watchman` and `nsfw`. Not sure which one is better.

Reply:
Let's move to a better package in the next PR.